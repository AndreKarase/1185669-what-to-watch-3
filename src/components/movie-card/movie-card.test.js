import React from 'react';
import renderer from 'react-test-renderer';
import MovieCard from './movie-card.jsx';

const movie = {
  title: ``,
  previewImage: ``,
  posterImage: ``,
  backgroundImage: ``,
  genre: ``,
  releaseDate: 1998,
  preview: ``,
  description: ``,
  rating: 10,
  scoresCount: 1,
  director: ``,
  starring: [``, ``, ``],
};

it(`Should render MovieCard`, () => {
  const tree = renderer.create(
      <MovieCard
        movie = {movie}
        onMouseEnter = {() => {}}
        onMouseLeave ={() => {}}
        onHeaderClick = {() => {}}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
