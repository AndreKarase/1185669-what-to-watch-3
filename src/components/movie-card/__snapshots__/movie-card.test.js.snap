// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render MovieCard 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onClick={[Function]}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <div
    className="small-movie-card__image"
  >
    <img
      alt=""
      height="175"
      src=""
      width="280"
    />
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
      >
        
      </a>
    </h3>
  </div>
</article>
`;
